workflow/rules/preprocess/create_overall_bbox.smk

```diff
@@ -3,10 +3,8 @@ rule create_overall_bbox:
     conda: "../../../environment.yml"
     input:
         "{OUTPUT_DIR}/input/{DATASET}.osm.pbf",
```

Comment:
`{DATASET}_filter-{FILTER}.osm.pbf`?

Reply:
I didn't think I'd changed the naming scheme with regard to filtering. What would be rationale be for this change?

Reply:
[pasting from orphaned comment] Ah, this was my quick guess when I saw the tests failing but you were still pushing commits. Feel free to ignore.